processors/ratelimiter/config.go

```diff
@@ -0,0 +1,27 @@
+package ratelimiter
+
+import (
+	"go.opentelemetry.io/collector/config"
+)
+
+// Config defines config for ratelimit processor.
+// The processor calls rate limit service for group of spans.
+// The processor either drops or forwards data based on rate limit response.
+// The tenant ID header is obtained from the context object.
+// The processor run immediately after tenantId processor
+type Config struct {
+	config.ProcessorSettings `mapstructure:"-"`
+
+	// RateLimitServiceHost defines host where rate limiter service is running default "localhost".
+	RateLimitServiceHost string `mapstructure:"service_host"`
```

Comment:
No need for the `RateLimit` prefix for these config fields.

Reply:
Updated accordingly